nuget/chocolatey/tools/chocolateysetup.psm1

```diff
@@ -700,4 +701,17 @@ param(
   }
 }
 
+function Run-Chocolatey {
+	Write-Debug "Running Chocolatey..."

```

Comment:
Do we call write-debug elsewhere in here or does it have another name?

Reply:
Found one at https://github.com/chocolatey/choco/pull/1507/files#diff-98d6abdebfea2b377e7a6d36d36c4cfaR591

Reply:
Yeah, I borrowed `Write-Debug` from elsewhere 😄